paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -136,7 +136,8 @@ CpuPassStrategy::CpuPassStrategy() : PassStrategy({}) {
                   "is_test_pass",                  //
                   // following two passes should be located in the last, since
                   // they will work on all fused ops.
-                  "expected_kernel_cache_pass",  //
+                  "expected_kernel_cache_pass",                //
+                  "fuse_reshape_transpose_scale_matmul_pass",  //
```

Comment:
why add last?

this is a very big fuse, maybe should be earlier.

Reply:
Now Bingyang are ready to re-implement the pass in future. This PR will be aborted.